client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -601,6 +609,43 @@
         };
     }
 
+    PackagingEmbeddedDirective.$inject = ['packages'];
+    function PackagingEmbeddedDirective(packages) {
+        return {
+            templateUrl: 'scripts/superdesk-packaging/views/packaging.html',
+            require: '^sdAuthoringContainer',
+            scope: {
+                item: '='
+            },
+            link: function(scope, elem, attrs, authoringCtrl) {
+                scope.$watch('item', function(item) {
+                    if (item && item.lockIt){
+                        scope.lock();
+                    } else {
+                        scope.origItem = null;
+                        scope.$applyAsync(function() {
```

Comment:
I agree, it should be a separate task - can you please open a ticket so that we don't forget about it? And we can then simply consider this one as "won't fix" in the context of this PR.